addons/l10n_it_edi_sdicoop/models/account_edi_format.py

```diff
@@ -136,7 +137,7 @@ def _l10n_it_post_invoices_step_1(self, invoices):
             })
             invoice.l10n_it_edi_attachment_id = attachment
 
-            if invoice._is_commercial_partner_pa():
+            if len(invoice.commercial_partner_id.l10n_it_pa_index or '') == 6:
```

Comment:
@dbkosky Why this change?

Reply:
Sorry I think some of these were a result of rebasing on the wrong branch